woof-code/rootfs-skeleton/usr/sbin/sfs_load

```diff
@@ -62,10 +68,10 @@ if [ -L /bin -o -L /sbin -o -L /lib -o -L /lib64 -o -L /lib32 -o -L /libx32 ]; t
   
    FLD_INVALID=""
   
-    for fld2 in bin sbin lib lib64 lib32 libx32
+    for fld2 in $sym_list
     do
     if [ -d /mnt/$MNTFLD/$fld2 -a ! -L /mnt/$MNTFLD/$fld2 ]; then
```

Comment:
The logic here is flawed, if I understand it correctly. If /libx32 is a symlink but devx/libx32 doesn't exist, sfs_load tells the user the devx SFS is "invalid".

Reply:
I misread this condition, I understood as if it's `if [ -d $fld2 -a ! -L /mnt/$MNTFLD/$fld2 ]; then`, but my point still stands, in a slightly different way. sfs_load needs to check if both $fld2 and /mnt/$MNTFLD/$fld2 are symlinks, both are directories, or at least one of them is missing (but deal correctly with the corner case where `! -e` is true because it's a broken symlink, and `! -L` when it's missing).